java/anaplan-connect-library/api-core/src/main/java/com/anaplan/client/Utils.java

```diff
@@ -305,4 +319,80 @@ public static void checkTarget(final File target, String path, String file, bool
     }
   }
 
+  /**
+   * Get absolute path if the path is relative
+   * @param target the path to check
+   * @return absolute path
+   */
+  public static Path getAbsolutePath(Path target) throws FileNotFoundException {
+    if (target == null) {
+      throw new FileNotFoundException("Path does not exist");
+    }
+    if (!target.isAbsolute()) {
+      target = Paths.get(target.toFile().getAbsolutePath());
+    }
+    return target;
+  }
+
+  public static byte[] createHash(String clientId) {
```

Comment:
Add Java docs for Public methods please for all classes.

Reply:
Java DC have been added across all public methods, please check latest push